rpc/detector/service.proto

```diff
@@ -0,0 +1,55 @@
+syntax = "proto3";
+
+package trivy.detector;
+option go_package = "detector";
+
+service OSDetector {
+    rpc Detect(OSDetectRequest) returns (DetectResponse);
+}
+
+message OSDetectRequest{
```

Comment:
Here and other places in `service.proto` you might want to reformat the file

Reply:
I introduced a plugin of GoLand to format a proto file, but it appears to be insufficient...I will introduce clang-format. Thanks.

Reply:
I added two commands to Makefile.